vkubelet/pod.go

```diff
@@ -217,6 +239,13 @@ func (s *Server) updatePodStatuses(ctx context.Context) {
 	span.AddAttributes(trace.Int64Attribute("nPods", int64(len(pods))))
 
 	for _, pod := range pods {
+		select {
```

Comment:
check it here?
https://github.com/virtual-kubelet/virtual-kubelet/blob/2743c07222211bfc5bd855b0e4b4732dc38f638d/vkubelet/vkubelet.go#L108

Reply:
Fixed